src/lozad.js

```diff
@@ -55,6 +58,17 @@ export default function (selector = '.lozad', options = {}) {
         load(elements[i])
         markAsLoaded(elements[i])
       }
+    },
+    triggerLoad() {
+      const elements = document.querySelectorAll(selector)
```

Comment:
@thiamsantos possible to get `element` as an argument? that way loading a selected element will be possible. So the function definition should look like:
```js
if (!isLoaded(element) {
 load(element)
 markAsLoaded(element)
}
```